packages/plugin-ocr-convert/src/server/providers/tencent-cloud.ts

```
@@ -0,0 +1,159 @@
+
+      case 'VehicleLicenseOCR':
+        response = await this.client.VehicleLicenseOCR(params);
+        return [
+          {
+            text: JSON.stringify(response),
+            original: response,
+          },
+        ];
+
+      case 'DriverLicenseOCR':
+        response = await this.client.DriverLicenseOCR(params);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)